core/platform/users/gae_current_user_services_test.py

```diff
@@ -0,0 +1,98 @@
+# coding: utf-8
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#Tests for the Gae current user services.
```

Comment:
Please do not comment this statement. Instead use triple double quotes for this, `"""` since this is a part of the documentation. Therefore, change this to: `"""Tests for the GAE current user services."""`

Reply:
Completed..!